### PR TITLE
PR: Content migration - Post CSS

### PR DESCRIPTION
**Pull request by AllUsernamesHaveBeenTaken**
#8839

### CODE REVIEW

File: docs/docs/component-css.md [hunk 1]
```diff
@@ -37,4 +37,6 @@ Following the same logic, you have the ability of creating multiple `module.css`
 
 You may take a look at a more in-depth [Component CSS tutorial](https://www.gatsbyjs.org/tutorial/part-two/#component-css) if you would like to see an example with explanation of multiple complex React components utilizing multiple `module.css` files and why you may want to use CSS Modules in your next project.
 
+### Other resources
 
+- [Introduction to postcss](https://www.smashingmagazine.com/2015/12/introduction-to-postcss/)
```
Comment (amberleyromo) [inline docs/docs/component-css.md]: Was this intended to be added to the post-css page?

File: docs/docs/post-css.md [hunk 2]
```diff
@@ -2,7 +2,53 @@
 title: PostCSS
 ---
 
-This is a stub. Help our community expand it.
+PostCSS transforms extended syntaxes and features into modern, browser-friendly CSS. This guide will show you how to get started with Gatsby and PostCSS.
 
-Please use the [Gatsby Style Guide](/docs/gatsby-style-guide/) to ensure your
-pull request gets accepted.
+### Installation and Configuration
```
Comment (amberleyromo) [inline docs/docs/post-css.md]: I would add a very short note here like in the `Sourcing from Wordpress` doc that explains that from this point, the guide assumes that a Gatsby site is initialized: https://www.gatsbyjs.org/docs/sourcing-from-wordpress/#quick-start

File: docs/docs/component-css.md (hunk 1 above)
Comment (amberleyromo) [reply docs/docs/component-css.md]: Also, `postcss` => `PostCSS`

File: docs/docs/post-css.md [hunk 3]
```diff
@@ -2,7 +2,53 @@
...
+
+1.  Install the Gatsby plugin **gatsby-plugin-postcss**.
```
Comment (amberleyromo) [inline docs/docs/post-css.md]: Convenient to link the plugin title to the plugin source code, perhaps? https://github.com/gatsbyjs/gatsby/tree/master/packages/gatsby-plugin-postcss

File: docs/docs/post-css.md [hunk 4]
```diff
@@ -2,7 +2,53 @@
...
+
+`npm install --save gatsby-plugin-postcss`
+
+2.  Include the plugin in your `gatsby-config.js` file.
+
+```javascript:title=gatsby-config.js
+plugins: [`gatsby-plugin-postcss`],
+```
+
+3.  Write your stylesheets using PostCSS (.css files) and require or import them as normal.
```
Comment (amberleyromo) [inline docs/docs/post-css.md]: Can you add a very short example here of PostCSS syntax?

**Comment by AllUsernamesHaveBeenTaken** [comment]
@amberleyromo I made the changes you requested.

**Pull request merged**